tests/README.md

```diff
@@ -0,0 +1,75 @@
+# Tests
+This directory contains the tests for the project. Below is the list of test functions and required pytest plugins.
+
+## Test Functions
+### cloud/test_infer.py
+- test_infer function
+
+### cloud/test_export.py
+- test_export function
+
+### cloud/test_compile.py
+- test_compile function
+
+### cloud/test_execute.py
+- test_execute function
+
+## Required Plugins
+- `pytest`
+- `pytest-mock`
+
+You can install them using pip:
+```sh
+pip install pytest pytest-mock
+```
+Alternatively, if you have specefied these dependencies in your `setup.py` or `setup.cfg` , you can install them using the extras_require feature:

```

Comment:
this has been moved to `pyproject.toml` now

Reply:
Okay, replacing it now.